config/initializers/paper_trail.rb

```diff
@@ -0,0 +1 @@
+PaperTrail.config.track_associations = false
```

Comment:
Just a question. Why don't we want to track associations?

Reply:
Don't know